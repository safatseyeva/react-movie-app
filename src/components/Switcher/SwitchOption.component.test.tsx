import * as React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import SwitchOption from './SwitchOption.component';

describe('<SwitchOption /> component:', () => {
  test('should render and match snapshot', () => {
    const handleSwitcherChange = jest.fn();
    const { asFragment } = 
      render(
        <SwitchOption 
          option='title'
          optionId={0}
          active={true}
          handleSwitcherChange={handleSwitcherChange}
        />
      );
    expect(asFragment()).toMatchSnapshot();
  });
});
