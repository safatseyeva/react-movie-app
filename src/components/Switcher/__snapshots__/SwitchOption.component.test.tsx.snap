// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SwitchOption /> component: should render and match snapshot 1`] = `
<DocumentFragment>
  <button
    class="undefined uppercase undefined"
  >
    title
  </button>
</DocumentFragment>
`;
